server/tests/batch_comparison/test_multi_contest_batch_comparison.py

```diff
@@ -1197,3 +1197,82 @@ def test_multi_contest_batch_comparison_editing_contests_after_uploads(
         batch_tallies_status = jurisdiction["batchTallies"]["processing"]
         assert batch_tallies_status["status"] == "PROCESSED"
         assert batch_tallies_status["error"] is None
+
+def test_updating_contests_while_jurisdictions_file_is_being_processed(
+    client: FlaskClient,
+    election_id: str,
+    jurisdiction_ids,  # pylint: disable=unused-argument
+    contest_ids,  # pylint: disable=unused-argument
+    election_settings,  # pylint: disable=unused-argument
+    manifests,  # pylint: disable=unused-argument
+    batch_tallies,  # pylint: disable=unused-argument
```

Comment:
I don't think you need any of these fixtures to execute for this test. Might be good to review all the test cases you added and remove unneeded fixtures. But not a huge issue if you miss a spot

Reply:
[9fed584](https://github.com/votingworks/arlo/pull/2043/commits/9fed584c13723d2859bc8c78d250d7b5f5336e93)